src/components/search/search-box.js

```diff
@@ -282,7 +282,8 @@ SearchBox.propTypes = {
   inputId: PropTypes.string,
   narrow: PropTypes.bool,
   disableModal: PropTypes.bool,
-  site: PropTypes.string
+  site: PropTypes.string,
+  wasSearched: PropTypes.bool
```

Comment:
Did this sneak in here?

Reply:
Whoops! Thanks for catching that. Removed!